crates/storage_impl/src/errors.rs

```diff
@@ -143,6 +143,9 @@ impl RedisErrorExt for error_stack::Report<RedisError> {
                     key: Some(key.to_string()),
                 })
             }
+            RedisError::KvSoftKillMode => {
+                self.change_context(DataStorageError::RedisOperationDisabled)
```

Comment:
Why are we converting into this error here?

Reply:
will remove this, was added for another approach